### DISCUSSION TITLE
iPhone 15 pro max and iOS 18 beta - images not showing in Photoprism library

### DISCUSSION THREAD

**Discussion by demetri90** [photoprism/photoprism#4437] [General]
Build 240711-2197af848. Other images import and show up normally. Only images from my iPhone 15 Pro Max no longer show. I believe it's due to some new image format or feature that Apple changed to. Below is a sample picture. 

https://www.icloud.com/iclouddrive/0884bnQCq44mt3eJPVjpd0Cqg#IMG_0747

**Comment by graciousgrey** [comment]
Might be related to: https://github.com/photoprism/photoprism/issues/4439

**Reply by lastzero** [reply]
Please see the following issue, which explains how you can help us with testing so that we can be sure that upgrading `libheif` will not break HEIC images taken with older versions of iOS:
- https://github.com/photoprism/photoprism/issues/4439
